packages/shared/src/types/props/shared.ts

```diff
@@ -19,18 +19,15 @@ export enum SecondaryCtaType {
     ContributionsReminder = 'ContributionsReminderSecondaryCta',
 }
 
-export const secondaryCtaTypeSchema = z.enum([
-    'CustomSecondaryCta',
-    'ContributionsReminderSecondaryCta',
-]);
+export const secondaryCtaTypeSchema = z.nativeEnum(SecondaryCtaType);
```

Comment:
What is the difference between an enum and a nativeEnum? And why do we need to change it?

Reply:
nativeEnum allowed us to reference the TS enum instead of repeating the string values. We found that passing the `SecondaryCtaType` to `z.enum` didn't work -
https://github.com/colinhacks/zod/discussions/2125